site/content/exercises/makecode/robotics/E7/index.md

```diff
@@ -0,0 +1,32 @@
+---
+template: exercise
+title: Functions
+level: 3
+exercise: 7
+category: Robotics
+tags: ['MakeCode', 'Functions', 'CPX']
+description: Creating functions to perform calculations and group commands
+---
+
+- **What are functions?**
+  - Functions are fundamental building blocks of programs\.
+  - They let you name a set of commands\.
+  - Once defined\, a function can be re\-used over and over\.
+- **Examples of functions:**
+  - Add\( *x\, y* \)
+    - Take two numbers  *x*  and  *y* \, and return the sum  *x\+y*
+    - \(takes input and returns output\)
+  - TurnOn\( *x* \)
+    - Take a number between 0 and 9\, and turn on that LED
+    - \(takes input\, does not return output\)
+  - PartyTime\(\)
+    - Set all pixels to red\, play a tune
+    - \(doesn’t take input\, doesn’t return output\)
```

Comment:
I'm not familiar with this version of MD, I'm assuming this is an unordered list with bullets and some bold headers?